app/controllers/admin/watched_words_controller.rb

```diff
@@ -41,7 +41,8 @@ def upload
             watched_word = WatchedWord.create_or_update_word(
               word: row[0],
               replacement: has_replacement ? row[1] : nil,
-              action_key: action_key
+              action_key: action_key,
+              case_sensitive: cast_to_boolean(row[2])
```

Comment:
Any particular reasons you didn't use `!!ActiveModel::Type::Boolean.new.cast(row[2])` provided by Rails?

Reply:
It appears to return `true` for values that are not necessarily true. I wanted to avoid creating case-sensitive Watched Words from incorrect CSV uploads.

I'm aiming for minimal impact on how Watched Words upload currently works.

However, the most common issue I foresee with the uploads will be whitespace related. I can resolve this by stripping the value and using `ActiveModel::Type::Boolean.new.cast`.  What do you think?

Reply:
Great. That will do.